index.js

```diff
@@ -19,5 +21,6 @@ wabbitzzz.Queue = queue();
 wabbitzzz.request = request();
 wabbitzzz.response = response();
 wabbitzzz.rpc = rpc();
+wabbitzzz.Connection = connection(opt);
```

Comment:
kill `opt` here, and maybe the `Connection` altogether -- you can't pass in opt here because these objects come out of the `require`, not `require()(opt)`

Reply:
my bad!

Reply:
oh ya, bad c/p